### DISCUSSION TITLE
LBB DV belum mendapatkan feedback

### DISCUSSION THREAD

**Discussion by DeanAlexander27** [teamalgoritma/community#593] [Q&A] [answered]
Dear Ka Instruktur

sebelumnya mohon maaf karena mengumpulkan telat 1 hari dari due date.
mohon apakah sudah ada feedback untuk LBB ini terima kasih..

![image](https://user-images.githubusercontent.com/123348110/224629039-f3d7e244-c083-4614-aaed-996728c59ff1.png)

**Comment by kevbow** [comment, marked as answer]
Dear @DeanAlexander27,

Mohon untuk menunggu terkait hal ini, tugas LBB Anda akan kami periksa segera. Mohon maaf atas ketidaknyamanannya, terima kasih.

**Reply by DeanAlexander27** [reply]
Dear Ka @kevbow 
terima kasih ka atas responnya, sudah done dengan mba Kinan..
terima kasih...
